go/store/nbs/store.go

```diff
@@ -1037,6 +1037,34 @@ func (nbs *NomsBlockStore) HasMany(ctx context.Context, hashes hash.HashSet) (ha
 	return nbs.hasMany(toHasRecords(hashes))
 }
 
+func (nbs *NomsBlockStore) hasManyInSources(ctx context.Context, srcs []hash.Hash, hashes hash.HashSet) (hash.HashSet, error) {
```

Comment:
Is the ctx future proofing for when we might need it? No used currently. If you think the interface needs it, I'd drop the name and make it '_' so people know that this is intentional

Reply:
I just didn't realize it was unneeded. Removed! Thanks for that :).